.coveragerc

```diff
@@ -1,2 +1,4 @@
-[run]
-omit = *migrations*
+[report]
+omit= 
+    *migrations*
+    *venv*
```

Comment:
Can the virtual environment be created outside the scope to ensure that this file will not be bloated by the many different virtual environment names?